src/main/groovy/org/mockito/release/internal/gradle/DefaultReleaseNotesPlugin.java

```diff
@@ -95,4 +119,9 @@ private static void configureNotes(DefaultReleaseNotesExtension notes, Project p
 
         //TODO make use of: ext.gh_writeAuthTokenEnvName = "GH_WRITE_TOKEN"
     }
+
+    private static File getTemporaryReleaseNotesFile(Project project){
```

Comment:
Why not use the "build" dir (e.g. project.getBuildDir())? If you don't want to use the build dir so that the data survives "clean", use ".gradle/mockito-release-tools/". 

".gradle" folder is guaranteed to exist and it's a nice place, already included in .gitignore for all Gradle-like projects.

I vote for using "build" dir for now (because it guarantees garbage collection) and perhaps use something else later on.